mobile/src/lib/background.js

```diff
@@ -28,18 +27,22 @@ const launchIosBackground = () => {
 
       logException('C\'est moi le Service Background!!!')
 
-      if (canStartBackup(store.getState)) {
+      if (store.getState().mobile.settings.backupImages && backupAllowed(getState().mobile.settings.wifiOnly)) {
```

Comment:
There is something weird in the call of `store.getState()` and the `getState()` in the two conditions.